packages/site/src/content/AnimatedPresence/AnimatedPresence.props.json

```diff
@@ -0,0 +1,49 @@
+[
```

Comment:
I've set up the git attribute file (in the feedback branch). TIL! Thanks for that. Let me know if it needs any tweaking.

Also, we should set up that diff check. Sounds like something you know how to do already, so want to pair for a few minutes when you get started and we can set it up together in the feedback branch instead of me bumbling around for 30 minutes in the documentation?

Reply:
I just moved the file! Should be in the right spot now.